libs/datasets/combined_datasets.py

```diff
@@ -293,23 +289,7 @@ def _build_data_and_provenance(
     datasource_dataframes: Mapping[str, pd.DataFrame],
     override=Override.BY_TIMESERIES,
 ) -> Tuple[pd.DataFrame, pd.DataFrame]:
-    # These are columns that are expected to have a single value for each FIPS. Get the columns
-    # from every row of each data source and then keep one of each unique row.
-    preserve_columns = [
-        CommonFields.AGGREGATE_LEVEL,
-        CommonFields.STATE,
-        CommonFields.COUNTRY,
-        CommonFields.COUNTY,
-    ]
-    all_identifiers = pd.concat(
-        df.reset_index().loc[
-            :, [CommonFields.FIPS] + list(df.columns.intersection(preserve_columns))
-        ]
-        for df in datasource_dataframes.values()
-    ).drop_duplicates()
-    # Make a DataFrame with a unique FIPS index. If multiple rows are found with the same FIPS then there
-    # are rows in the input data sources that have different values for county name, state etc.
-    fips_indexed = all_identifiers.set_index(CommonFields.FIPS, verify_integrity=True)
+    fips_indexed = fips_index_geo_data(pd.concat(datasource_dataframes.values()))
```

Comment:
Can you call this `dataset_utils.fips_index_geo_data`?

Reply:
Done in 5c7c0f0